src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -1834,7 +1834,7 @@ public final class JavadocTokenTypes {
      * Area tag name.
      *
      * <p><b>Example:</b></p>
-     * <pre>{@code &lt area shape="rect" &gt}</pre>
+     * <pre>{@code < area shape="rect" >}</pre>
```

Comment:
This update means that if we put `{@code` after each `<pre>` we can remove escaping at all so code will be easy to read in sources and it rendering well html.
Can you update all to have it ? And we will close this madness on escaping